sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -126,4 +126,10 @@ object StaticSQLConf {
       .intConf
       .createWithDefault(1000)
 
+  val MAX_BROADCAST_EXCHANGE_THREADNUMBER =
+    buildStaticConf("spark.sql.broadcastExchange.maxThreadNumber")
+      .doc("MAX number of threads can hold by BroadcastExchangeExec which controls" +
```

Comment:
Still, users won't know what's `BroadcastExchangeExec` and `MAX number of threads`. Please do your diligence for this doc. Also, we might have to describe when to increase and when to decrease.